src/main/java/org/isf/users/mapper/UserMapper.java

```diff
@@ -19,15 +19,16 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
-package org.isf.menu.mapper;
+package org.isf.users.mapper;
 
-import org.isf.menu.dto.UserDTO;
+import org.isf.users.dto.UserDTO;
```

Comment:
shouldn't this import be found later in the list?   They are not in alphabetic order.